main.c

```diff
@@ -34,16 +34,16 @@ enum FLIGHT_PHASE {
 };
 
 enum FLIGHT_PHASE state = PRE_FLIGHT;
-const uint32_t BOOST_LENGTH_MS = 1000; // for the purposes of debugging
-const uint32_t COAST_LENGTH_MS = 2000000; // see above
+const uint32_t BOOST_LENGTH_MS = 9000; 
+const uint32_t COAST_LENGTH_MS = 35000-9000;
```

Comment:
I feel like this should be `35000 - BOOST_LENGTH_MS`

Reply:
true i thought abt doing that dunno why i didnt